American Whitewater/Extensions/CLLocationCoordinate2D+difference.swift

```diff
@@ -0,0 +1,9 @@
+import CoreLocation
+
+extension CLLocationCoordinate2D {
+    /// Checks if either latitude or longitude has changed by more than a certain amount
```

Comment:
Is there some sort of docstring style we should be using instead?

Reply:
This is it, the extra slash indicates a doc comment (or apparently /** ... **/ too? I always just use the /// form). Beyond that I don't have an opinion but if you want to set a style for what should be in the comments I'm happy to use it.